src/js/components/Meter/Meter.js

```diff
@@ -19,9 +20,12 @@ const deriveMax = (values) => {
 const Meter = forwardRef(
   (
     {
+      'aria-label': ariaLabel,
       background = { color: 'light-2', opacity: 'medium' },
       color,
       direction = 'horizontal',
+      max,
```

Comment:
Was `max` documented, but not an actual prop?

Reply:
Are there other props documented but not represented as props here?

Reply:
yes there are other props documented but not represented here I had pulled out the `max` so I could use that in the message since that was just being passed in the `...rest`